quickstarts/atlassian/atlassian-jira/atlassian-jira.yml

```diff
@@ -0,0 +1,47 @@
+# Name of the quickstart (required)
```

Comment:
does the `id` get generated automatically? [reference](https://github.com/newrelic/newrelic-quickstarts/blob/main/quickstarts/aws/aws-api-gateway/config.yml#L1)

Reply:
The `id` of the quickstart will autogenerated, but the `id` of the install plan is user defined